test/suite/test_import03.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_import03.py
+# Import a table into a running database.
+
+import os, re, shutil
+import wiredtiger, wttest
+from wtscenario import make_scenarios
+
+def timestamp_str(t):
+    return '%x' % t
+
+class test_import03(wttest.WiredTigerTestCase):
+    conn_config = 'cache_size=50MB,log=(enabled),statistics=(all)'
+    session_config = 'isolation=snapshot'
+    scenarios = make_scenarios([
+        # Importing into a new database.
+        ('new', dict(import_type='new')),
+        # Importing into an existing database with other files.
+        ('existing', dict(import_type='existing')),
+        # Importing into the same database. We should expect a failure.
+        ('same', dict(import_type='same')),
+    ])
+
+    def update(self, uri, key, value_tuple, commit_ts):
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction()
+        cursor.set_key(key)
+        cursor.set_value(*value_tuple)
+        cursor.insert()
+        self.session.commit_transaction('commit_timestamp=' + timestamp_str(commit_ts))
+        cursor.close()
+
+    def check(self, uri, key, value_tuple, read_ts):
+        # Read the entire record.
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction('read_timestamp=' + timestamp_str(read_ts))
+        cursor.set_key(key)
+        self.assertEqual(0, cursor.search())
+        self.assertEqual(list(value_tuple), cursor.get_value())
+        self.session.rollback_transaction()
+        cursor.close()
+
+        # Now let's test something table specific like a projection.
```

Comment:
We only need to support simple tables for this project.

Reply:
It tests that the table specific configuration (not `file_metadata`) is getting used properly.
It works so I don't see a reason to remove this.